sormas-e2e-tests/src/test/resources/JsonFiles/Person-Post.json

```diff
@@ -0,0 +1,7 @@
+{
+
+  "uuid": "RKVAOM-ZNAAFU-R2KF6Z-6BENKHEY",
+  "pseudonymized": true,
+  "firstName": "Alina",
```

Comment:
let's try to avoid using our names. Also, is not a good approach to use the same name, with the same data each time we're running a test.

Reply:
:) addressed. And it is overwritten in during the execution with some generated but recognizable names.